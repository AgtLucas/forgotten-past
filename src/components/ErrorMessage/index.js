import React from 'react';
import styles from './styles.module.css';

const ErrorMessage = ({ children, ...rest }) => (
  <span
    className={styles.error}
    aria-live='polite'
    {...rest}
  >
    {children}
  </span>
);

export default ErrorMessage;