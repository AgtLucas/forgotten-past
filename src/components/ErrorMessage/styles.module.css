.error {
  font-size: 14px;
  color: var(--red);
}