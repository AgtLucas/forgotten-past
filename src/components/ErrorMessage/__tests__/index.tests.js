import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import ErrorMessage from '../';

describe('ErrorMessage', () => {
  it('should render ErrorMessage', () => {
    const wrapper = shallow(<ErrorMessage>Error</ErrorMessage>);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});