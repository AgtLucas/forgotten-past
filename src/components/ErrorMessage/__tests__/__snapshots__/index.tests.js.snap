// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMessage should render ErrorMessage 1`] = `
<span
  aria-live="polite"
  className="error"
>
  Error
</span>
`;
